lib/nokogiri/xml/parse_options.rb

```diff
@@ -186,21 +425,88 @@ def #{constant.downcase}?
         RUBY
       end
 
+      # :call-seq:
+      #   strict
+      #
+      # Turns **off** option `recover`:
+      #
+      # ```
+      # options = ParseOptions.new.recover.compact.big_lines
+      # # => #<Nokogiri::XML::ParseOptions: ... recover, compact, big_lines>
+      # options.strict
+      # # => #<Nokogiri::XML::ParseOptions: ... strict, compact, big_lines>
+      # ```
       def strict
         @options &= ~RECOVER
         self
       end
 
+      # :call-seq:
+      #   strict?
+      #
+      # Returns whether option `strict` is **on**:
+      #
+      # ```
+      # options = ParseOptions.new.recover.compact.big_lines
+      # # => #<Nokogiri::XML::ParseOptions: ... recover, compact, big_lines>
+      # options.strict? # => false
+      # options.strict
+      # # => #<Nokogiri::XML::ParseOptions: ... strict, compact, big_lines>
+      # options.strict? # => true
+      # ```
       def strict?
         @options & RECOVER == STRICT
       end
 
+      # :call-seq:
+      #    self == object
+      #
+      # Returns whether `object` is equal to `self` (`object.to_i == to_i`):
```

Comment:
I think this is a confusing statement, we can't both test for identity (`self`) and the value of the integer.

I would describe this as "Returns true if the same options are set in `self` and `other`."